PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,9 @@
 - [ ] `./mach test-tidy` does not report any errors
 - [ ] These changes fix #__ (github issue number if applicable).
 
+<!-- If you add new tests for wpt -->
+- [ ] `./etc/ci/manifest_changed.sh` does not report any errors
```

Comment:
How about putting this checkbox under `There are tests for these changes`?

For example,

```
<!-- Either: -->
- [ ] There are tests for these changes
  - [ ] `./etc/ci/manifest_changed.sh` does not report any errors

<!-- OR -->
- [ ] These changes do not require tests because _____
```

Reply:
Put `Either` is good, but we need to mention that `./etc/ci/manifest_changed.sh` is only for WPT

Reply:
@jdm oh! wow! Sounds a good solution for me!

Reply:
Sounds good.